src/expressions/dataTypes/casting/tryCastToType.ts

```diff
@@ -59,6 +61,8 @@ function castToPrimitiveType(from: ValueType, to: ValueType): (value: Value) =>
 			return castToDecimal(instanceOf);
 		case ValueType.XSINTEGER:
 			return castToInteger(instanceOf);
+		case ValueType.XSNUMERIC:
```

Comment:
Maybe a comment explaining why this is here? Like how you explained it in Slack

Reply:
Will do.